addons/sale_expense/tests/test_reinvoice.py

```diff
@@ -226,14 +229,14 @@ def test_expenses_reinvoice_case_2_reset_sheet_to_draft(self):
 
         self.assertRecordValues(self.expense_sale_order.order_line, [
             # [0] Line not created from a re-invoiced, should never be changed
-            {'qty_delivered': 0.0, 'product_uom_qty': 3.0, 'name': 'expense_employee: expense_1 invoicing=order, expense=sales_price'},
+            {'qty_delivered': 0.0, 'product_uom_qty': 3.0, 'expense_ids': []},
```

Comment:
Looks like this was linked to `sale_exp_order_sale_1`?
Isn't this supposed to be a refactoring?